engine/src/tangl/mechanics/sandbox/incremental.py

```
@@ -0,0 +1,237 @@
+    if not isinstance(additions, list):
+        return None
+    caller.locals["_sandbox_incremental_fragments"] = []
+    return additions or None
+
+
+__all__ = [
+    "process_sandbox_incremental_game_move",
+    "project_sandbox_incremental_game_moves",
+    "render_sandbox_incremental_journal",
+    "reconcile_incremental_games_on_sandbox_tick",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)